### ISSUE TITLE
Improve Intellisense

### ISSUE THREAD

**Issue by mattjohnsonpint** [jbevain/EmojiVS#3]
When Intellisense works (see #2), it's still not quite what I expect.

I have to type at least `// :` and press `ctrl-space` to see the dropdown.

It would be much nicer if the dropdown showed up as soon as I enter a `:` (as long as I'm in a comment).

**Comment by jbevain** [comment]
Yeah I made it this way because currently we do not detect that we're in a comment, and we even allow emojis outside of comments. That needs to be fixed first. Otherwise you're going to get prompted whenever you use a C# ternary expression :)

**Comment by TimChen44** [comment]
I use the Chinese system, "Ctrl-space" is the input method Toggle shortcut key, shortcut key conflicts with intelligence tips.

Can I customize a shortcut?
